tests/comptroller/utils.ts

```diff
@@ -0,0 +1,173 @@
+import { newMockEvent, newMockEventWithParams } from "matchstick-as/assembly/index"
+import { Address, ethereum, Bytes, BigInt, BigDecimal } from "@graphprotocol/graph-ts"
+
+import { 
+  Comptroller,
+  Market,
+  Account,
+} from "../../generated/schema"
+
+import { MarketListed } from "../../generated/Comptroller/Comptroller"
+import {
+  Mint,
+  Transfer,
+  Redeem,
+  Borrow,
+  RepayBorrow,
+} from "../../generated/templates/CToken/CToken"
+
+import { handleMarketListed } from "../../src/comptroller"
+
+import {
+  handleMint,
+  handleTransfer,
+  handleRedeem,
+  handleBorrow,
+  handleRepayBorrow,
+} from "../../src/ctoken"
+
+export function saveComptroller(): void {
+  let comptroller = Comptroller.load('1')
+  if (!comptroller) {
+    comptroller = new Comptroller('1')
+    comptroller.priceOracle = "0x892be716dcf0a6199677f355f45ba8cc123baf60"
+    comptroller.liquidationIncentive = BigDecimal.fromString("1.1")
+    comptroller.closeFactor = BigDecimal.fromString("0.5")
+    comptroller._markets = []
+  }
+  comptroller.save()
+}
+
+export function saveAccount(address: string): void {
+  let account = Account.load(address)
+  if (!account) {
+    account = new Account(address)
+    account.tokens = []
+    account.countLiquidated = 0
+    account.countLiquidator = 0
+    account.hasBorrowed = false
+  }
+  account.save()
+}
+
+export function createMarketListedEvent(address: string): MarketListed {
+  let newMarketListed = changetype<MarketListed>(newMockEvent());
+  newMarketListed.parameters = new Array();
+  let mTokenParam = new ethereum.EventParam("mToken", ethereum.Value.fromAddress(Address.fromString(address)));
+
+  newMarketListed.parameters.push(mTokenParam);
+  return newMarketListed;
+}
+
+export function handleMarketListeds(events: MarketListed[]): void {
+  events.forEach(event => {
+    handleMarketListed(event)
+  })
+}
+
+export function createNewMintEvent(from: string, to: string, amount: BigInt, hash: string): Mint {
+  let minterParam = new ethereum.EventParam("minter", ethereum.Value.fromAddress(Address.fromString(from)));
+  let mintAmountParam = new ethereum.EventParam("mintAmount", ethereum.Value.fromUnsignedBigInt(amount));
+  let mintTokensParam = new ethereum.EventParam("mintTokens", ethereum.Value.fromUnsignedBigInt(amount));
+
+  let newMint = changetype<Mint>(newMockEventWithParams([minterParam, mintAmountParam, mintTokensParam]));
+  newMint.address = Address.fromString(to); // Set the address of the event to the mToken address
+  newMint.transaction.hash = Bytes.fromHexString(hash) as Bytes; // Set the hash of the transaction
+  return newMint;
+}
+
+export function handleNewMints(events: Mint[]): void {
+  events.forEach(event => {
+    handleMint(event)
```

Comment:
We're definitely covering `handleMint` - not sure what is up with code coverage but I'll troubleshoot more.